internal/replay.go

```diff
@@ -104,15 +104,8 @@ func (r *ReplayCmd) RunReplays(cmd *cobra.Command, replays []ReplayConfig) (fail
 	for i, replay := range replays {
 		cmd.Println(colorstring.Color(fmt.Sprintf(
 			"[cyan][%d/%d][reset] SLO: %s, Project: %s, From: %s, To: %s",
-			i+1,
-			len(replays),
-			replay.SLO,
-			replay.Project,
-			replay.From.Format(
-				flags.TimeLayout,
-			),
-			time.Now().In(replay.From.Location()).Format(flags.TimeLayout),
-		)))
+			i+1, len(replays), replay.SLO, replay.Project,
+			replay.From.Format(flags.TimeLayout), time.Now().In(replay.From.Location()).Format(flags.TimeLayout))))
```

Comment:
why did you change the formatting? please revert this

Reply:
No, I didn't :confused: